addons/l10n_mn/migrations/1.1/post-migrate.py

```diff
@@ -15,7 +15,7 @@ def migrate(cr, version):
             if matching_template:
                 cashflow_tag = matching_template.tag_ids
                 if cashflow_tag:
-                    account.write({'tag_ids': [Command.link(cashflow_tag.id)]})
+                    account.write({'tag_ids': [Command.link(cf.id) for cf in cashflow_tag]})
```

Comment:
I would tend to say that I prefer having it like this
```suggestion
                    account.write({'tag_ids': [Command.link(tag_id) for tag_id in cashflow_tag.ids]})
```